### PR TITLE
Fix #1471: Add Intellij idea settings to repo

### LINKED ISSUE #1471: Add Intellij idea settings to repo

### PR DESCRIPTION
**Pull request by harshach**
### Describe your changes :

I worked on the ..... because ...

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have performed a self-review of my own. 
- [ ] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<project version="4">
+  <component name="ExternalStorageConfigurationManager" enabled="true" />
+  <component name="MavenProjectsManager">
+    <option name="originalFiles">
+      <list>
+        <option value="$PROJECT_DIR$/pom.xml" />
+      </list>
+    </option>
+  </component>
+  <component name="ProjectRootManager" version="2" project-jdk-name="11" project-jdk-type="JavaSDK" />
```
Comment (ksivam) [inline .idea/misc.xml:11]: you might want to set `languageLevel="JDK_11"`

**Comment by ksivam** [comment]
shall we also add two files (apache.xml and profiles_settings.xml) to `.idea/copyright` 

**apache.xml**
```xml
<component name="CopyrightManager">
  <copyright>
    <option name="notice" value="Licensed to the Apache Software Foundation (ASF) under one or more&#10;contributor license agreements. See the NOTICE file distributed with&#10;this work for additional information regarding copyright ownership.&#10;The ASF licenses this file to You under the Apache License, Version 2.0&#10;(the &quot;License&quot;); you may not use this file except in compliance with&#10;the License. You may obtain a copy of the License at&#10;&#10;http://www.apache.org/licenses/LICENSE-2.0&#10;Unless required by applicable law or agreed to in writing, software&#10;distributed under the License is distributed on an &quot;AS IS&quot; BASIS,&#10;WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.&#10;See the License for the specific language governing permissions and&#10;limitations under the License. &#10;" />
    <option name="myName" value="apache" />
  </copyright>
</component>
```

**profiles_settings.xml**
```xml
<component name="CopyrightManager">
  <settings>
    <module2copyright>
      <element module="All" copyright="apache" />
    </module2copyright>
  </settings>
</component>
```

**Review by ksivam — APPROVED** [review]
just 2 comments, rest looks good.

**Comment by harshach** [comment]
Thanks @ksivam for the review. I'll add the copyRight element in another PR. Will merge this in.

**Pull request merged**